### ISSUE TITLE
Allow controlong line height

### ISSUE THREAD

**Issue by khaledhosny** [justvanrossum/fontgoggles#70]
Many fonts have glyphs that would go beyond line height and currently these are cropped. It would be nice if line height can be manually set for cases like these (no cropping would be good to, and man be optionally using the glyph bounds instead of the line height info from the font).

دے  چكے ہين and Noto Nastaliq Urdu.

![](https://user-images.githubusercontent.com/93914/77262014-66e16680-6c9b-11ea-8da5-01e590dec0ee.png)

**Comment by punchcutter** [comment]
I agree 100%. At least half the things I work on can use this. It would also be useful if there were a visible line for the bounding box to be totally clear where things will get cut off. I test a lot in browsers and for certain scripts the default line-height needs to be set quite high so having that option here would be very useful.

**Comment by justvanrossum** [comment]
You can more or less do what you want by going to the "Options" tab and reduce the "Relative Size" and fiddle with the "Baseline" slider.

Zooming with pinch zoom, or option-scroll or command-plus/minus changes the size of the "box".

Unfortunately, with the way the UI is built, it is currently not possible to not do cropping.

All of this together should certainly be reconsidered in a future version.

**Comment by khaledhosny** [comment]
I missed the Option tab. That is good enough for me, thanks.

**Comment by justvanrossum** [comment]
I will leave the issue open anyway as a reminder of things to consider if/when a bigger redesign is to be done. The fact that you missed the Options tab is already reason enough to consider this part of the UI suboptimal...

**Comment by ohbendy** [comment]
I'd find it nice if the line height could be adjusted too :-) On my large external display, I need to crank up the font size to 100 to see things clearly, and that makes tall/deep clusters disappear outside the preview. I can fiddle the baseline up and down, but you can see it's still clipping the top marks here:

![Screenshot 2023-02-07 at 10 29 42](https://user-images.githubusercontent.com/12471463/217220243-fa6117d2-889f-4d8d-941f-dfae1845c0b0.png)

**Comment by justvanrossum** [comment]
@ohbendy: try command + and - for "normal" zoom, or zoom gestures for that matter. Then you can keep the "relative size" small to accomodate a better line height.

**Comment by ohbendy** [comment]
Oh yay, that's exactly it! Sorry for the needless poking :)
